### DISCUSSION TITLE
[Feature] Send an email to users you have shared an album with

### DISCUSSION THREAD

**Discussion by anoadragon453** [immich-app/immich#7442] [Feature Request]
### The feature

I'd like to send email notifications to my family members whenever I share an album with them. Currently I have to message everyone that I share an album with, which gets quite exhausting after each time.

The emails themselves don't need to be particularly complex. Something as simple as "X has shared the album Y with you! Click here to view it" is enough.

This functionality could be extended to notifications of many other events in the app. But for now, shared album notifications are the most desirable for my use case.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1671

**Reply by anoadragon453** [reply]
I don't think this is a duplicate, though they are closely related. That discussion describes a notification being sent when an image is added to an album. Whereas this discussion describes a notification being sent when an album is shared.

**Comment by lieberpi** [comment]
Would it be possible to have a "Notifications" option in the settings where one could select which events would send a notification?

For example:
- An album has been shared with you
- A new picture/video has been added to an album that has been shared with you
- A new comment has been made to a picture in an album that has been shared with you
- Someone liked an image in an album that has been shared with you

This would be great!
